Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.py

```diff
@@ -1536,6 +1676,14 @@ def test_module():
         demisto.results(get_table_name_command())
     if demisto.command() == 'servicenow-get-ticket-notes':
         demisto.results(get_ticket_notes_command())
+    if demisto.command() == 'servicenow-items-query':
+        demisto.results(query_items())
+    if demisto.command() == 'servicenow-item-details-get':
+        demisto.results(get_item_command())
+    if demisto.command() == 'servicenow-item-order-create':
```

Comment:
Why not `servicenow-create-item-order`?

Reply:
same like before, i took the command name from here:
https://github.com/demisto/etc/issues/22557#issuecomment-603766282